internal/sink/sink.go

```diff
@@ -68,31 +67,32 @@ func NewSink(c Config, logger *zap.Logger, listener ancla.Register) Sink {
 			kafka := &Kafka{
 				id:         l.Registration.CanonicalName,
 				brokerAddr: k.BootstrapServers,
-				topic:      "test",
+				topic:      "quickstart-events",
```

Comment:
this is based off the kafka docs for testing locally... i imagine we will be including topic as a part of the kafka webhook struct that users will be sending us?

Reply:
yup